tools/nixpkgs-hammer

```diff
@@ -203,7 +203,7 @@ def render_code_location_ansi(loc: SourceLocation) -> str:
         pointer = " " * (column - 1) + "^"
 
         location_lines = [
-            f"Near {loc.file}:{loc.line}:{loc.column}:",
+            f"Near {loc.file}:{loc.line}:",
```

Comment:
Is this intentional?

Reply:
Oh whoops, that's not what I want. One sec.

Reply:
Okay,  fixed up better.